packages/ui/src/lib/theme/radius.ts

```diff
@@ -1,16 +1,17 @@
-type RadiusAlias = 'xs' | 'sm' | 'md' | 'lg' | 'xl'
-
-export const radius: Partial<Record<number | RadiusAlias, string>> = {
-  0: '8px',
-  1: '12px',
-  2: '16px',
-  3: '20px',
-  4: '28px',
+const base = {
+  0: 8,
+  1: 12,
+  2: 16,
+  3: 20,
+  4: 28,
 }
```

Comment:
```suggestion
} as const
```

By adding `as const` here we're changing the type of the `base` value from `number` to the actual value, for example `8` or `28`. 

Also noticed that we don't need to add `px` value for border radius 👍

Reply:
I just thought it's nice to be explicit but I'm all good if you think it's clear enough to without `px` :)